src/tls_schannel.c

```diff
@@ -319,12 +319,59 @@ int tls_start(tls_t *tls)
 
 	}
 
-	sbin[0].cbBuffer = len;
+    /* management of SEC_E_INCOMPLETE_MESSAGE event - start */
+    {
```

Comment:
Please, make proper indent for blocks. 2 nested blocks have the same indent as the outer `while` loop. Or just remove the block and use only comments to show where it starts and ends.